act/workers/scio2.py

```diff
@@ -236,23 +239,23 @@ def add_to_act(actapi: act.api.Act,
     report_mentions_fact(
         actapi,
         "threatActor",
-        set(doc.get("threatactor", {}).get("ThreatActors", [])),
+        {format_threat_actor(ta) for ta in doc.get("threatactor", {}).get("ThreatActors", [])},
         report_id,
         output_format)
 
     # Tools
     report_mentions_fact(
         actapi,
         "tool",
-        {tool.lower() for tool in doc.get("tools", {}).get("Tools", [])},
+        {format_tool(tool) for tool in doc.get("tools", {}).get("Tools", [])},
         report_id,
         output_format)
 
     # Sector
     report_mentions_fact(
         actapi,
         "sector",
-        set(doc.get("sectors", []).get("sectors", [])),
+        set(doc.get("sectors", {}).get("sectors", [])),
```

Comment:
This is a bug that would not cause any error as long as we had "sectors" in the scio document.